element_interface/intan_loader.py

```diff
@@ -248,40 +240,51 @@ def load_rhs(folder: str, file_expr: str):
             rhs_data["timestamps"] (np.array_like): Relative timestamps in seconds.
     """
 
+    rhs_data = {}
+    
+    # Get header
     header_filepath = next(Path(folder).glob("info.rhs"))
     with open(header_filepath, "rb") as fid:
-        header = read_header(fid)
+        rhs_data["header"] = _read_header(fid)
 
-    time_file = next(Path(folder).glob("time.dat"))
 
-    timestamps = (
+    # Get timestamps
+    time_file = next(Path(folder).glob("time.dat"))
+    
+    rhs_data["timestamps"] = (
         np.memmap(time_file, dtype=np.int32)
-        / header["frequency_parameters"]["amplifier_sample_rate"]
+        / rhs_data["header"]["frequency_parameters"]["amplifier_sample_rate"]
     )
 
-    rhs_data = dict(header=header, timestamps=timestamps, recordings={})
-
+    # Get data files
     file_paths = Path(folder).glob(file_expr)
-    file_paths = [x for x in file_paths if x.as_posix() != "time.dat"]
-
-    for file_path in file_paths:
-        file_path = file_path.as_posix()
-        if "amp" in file_path:
-            signal = np.memmap(file_path, dtype=np.int16)
+    
+    if file_expr == "*":
+        file_paths = list(folder.glob("[!info|!time|!Zone]*"))
+        
+    # Get recording data
+    rhs_data["recordings"] = {}
+    
+    for file_path in sorted(file_paths):
+
+        signal = np.memmap(file_path, dtype=np.int16)
```

Comment:
Not all data types are int16. Some of them are uint16.

Reply:
@tdincer you're right. Let me fix that.